com.github.fabiocolacio.marker.json

```diff
@@ -0,0 +1,60 @@
+{
+  "app-id": "com.github.fabiocolacio.marker",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.26",
+  "sdk": "org.gnome.Sdk",
+  "command": "com.github.fabiocolacio.marker",
+  "finish-args": [
+    /* X11 + XShm access */
+    "--share=ipc", "--socket=x11",
+    /* Wayland access */
+    "--socket=wayland",
+    /* We want full fs access so we can read the files */
+    "--filesystem=home"
```

Comment:
In theory this isn't needed, it only opens a single file at a time right?

Reply:
When i remove it i can't see any files in the open dialog

Reply:
Ah, Upstream should start using this API https://developer.gnome.org/gtk3/stable/gtk3-GtkFileChooserNative.html for its dialogs.

Reply:
Ok, i made a issue upstream https://github.com/fabiocolacio/Marker/issues/133

Reply:
It seams they did already implement this. (I didn't test it yet)
https://github.com/fabiocolacio/Marker/commit/489cec84a0b7932af4203dc3aacff50a8399530e